From bd9e614eb81c04506fb1163e2d90cf65a46afff1 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Sat, 5 Mar 2016 21:43:47 +0800
Subject: [PATCH] ARM64: dts: rockchip: add gt9xx touchscreen support

need to modify the i2c4 scl and sda connection in hardware.

Change-Id: I57981a3891047412b9ea61112fbf5ff4a23223b3
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3366-tb.dts | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3366-tb.dts b/arch/arm64/boot/dts/rockchip/rk3366-tb.dts
index 995a1ae22911..2221722e947a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3366-tb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3366-tb.dts
@@ -532,6 +532,22 @@
 	};
 };
 
+&i2c4 {
+	status = "okay";
+
+	gt9xx: gt9xx@14 {
+		compatible = "goodix,gt9xx";
+		reg = <0x14>;
+		touch-gpio = <&gpio5 11 IRQ_TYPE_LEVEL_LOW>;
+		reset-gpio = <&gpio5 10 GPIO_ACTIVE_HIGH>;
+		max-x = <1200>;
+		max-y = <1900>;
+		tp-size = <911>;
+		tp-supply = <&vcc_tp>;
+		status = "okay";
+	};
+};
+
 &i2s_8ch {
 	status = "okay";
 	rockchip,i2s-broken-burst-len;
-- 
2.35.3

